Classes/Issues/Request/IssueRequestModel.swift

```diff
@@ -42,6 +42,63 @@ final class IssueRequestModel: ListDiffable {
         self.date = date
         self.event = event
 
+        let styledString: StyledTextString
+        if actor == user {
+            styledString = IssueRequestModel.buildSelfString(actor: actor,
+                                                             user: user,
+                                                             date: date,
+                                                             event: event)
+        } else {
+            styledString = IssueRequestModel.buildString(actor: actor,
+                                                         user: user,
+                                                         date: date,
+                                                         event: event)
+        }
+
+        self.string = StyledTextRenderer(
+            string: styledString,
+            contentSizeCategory: contentSizeCategory,
+            inset: UIEdgeInsets(
+                top: Styles.Sizes.inlineSpacing,
+                left: 0,
+                bottom: Styles.Sizes.inlineSpacing,
+                right: 0
+            )
+        ).warm(width: width)
+    }
+
+    static private func buildSelfString(actor: String,
+                                        user: String,
```

Comment:
`user` is never used inside this function so no need to pass it in. Instead of passing in `actor` you may want to pass in `user` and switch the use of `actor` later on to `user`. I think it reads better as `user`